app/src/main/java/de/tum/in/tumcampusapp/services/GeofencingRegistrationService.kt

```diff
@@ -0,0 +1,88 @@
+package de.tum.`in`.tumcampusapp.services
+
+import android.annotation.SuppressLint
+import android.app.PendingIntent
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.support.v4.app.JobIntentService
+import android.support.v4.content.ContextCompat
+import com.google.android.gms.location.Geofence
+import com.google.android.gms.location.GeofencingClient
+import com.google.android.gms.location.GeofencingRequest
+import com.google.android.gms.location.LocationServices
+import de.tum.`in`.tumcampusapp.auxiliary.Const.GEOFENCING_SERVICE_JOB_ID
+import de.tum.`in`.tumcampusapp.auxiliary.Utils
+
+
+/**
+ * Service that receives Geofencing requests and registers them.
+ */
+class GeofencingRegistrationService : JobIntentService() {
+
+    private lateinit var mGeofencePendingIntent: PendingIntent
```

Comment:
why the `m` prefix? You are mixing it here. One member has it the other one does not. Please be consistent and remove it everywhere. The IDE colors members in a different way in any case - no added benefit of using it.